src/http/multipart.rs

```diff
@@ -8,13 +8,17 @@ use crate::internal::prelude::*;
 
 impl CreateAttachment<'_> {
     async fn into_part(self) -> Result<Part> {
-        let mut part = match self.data {
+        #[rustfmt::skip]
+        let Self { filename, data, description } = self;
+        drop(description);
```

Comment:
This should wait until after we've changed our rustfmt settings to include `struct_lit_single_line`.